specs/multispec.yml

```diff
@@ -120,6 +120,8 @@ matrix:
     - distros:
         - fedora-31-x86_64
         - fedora-34-x86_64
+        - rhel-7-x86_64
```

Comment:
This is for version 10 and not 9.6 as is written in PR title.

But only nitpick from my side.

Reply:
Ah right, rebasing the PR must have thrown it into a different version. Should be fixed now

Reply:
+1